LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2019 GitHub
```

Comment:
GitHub doesn't own this theme? Or do they. Seems like we're giving them copyright over our site.